test/mock/data/sample_raw_job.json

```diff
@@ -0,0 +1,38 @@
+{
+  "data": {
+    "end_date": "2022-12-05T12:42:27.164470",
+    "id": "0eea9b1b-f86b-4770-bd52-a8d357a21459",
+    "params": {
+      "calc_perf": true,
+      "calc_state": true,
+      "calc_traj": true,
+      "climate_gee_dataset": null,
+      "crs": "",
+      "geojsons": "[]",
+      "type": "Polygon",
+      "local_context": {
+        "area_of_interest_name": "finland-all-regions-buffer-10.000",
+        "base_dir": "/home/samweli/trends_earth_data"
+      },
+      "ndvi_gee_dataset": "users/geflanddegradation/toolbox_datasets/ndvi_modis_2001_2021",
+      "prod_mode": "TrendsEarth-LPD-5",
+      "prod_perf_year_final": 2015,
+      "prod_perf_year_initial": 2001,
+      "prod_state_year_bl_end": 2012,
+      "prod_state_year_bl_start": 2001,
+      "prod_state_year_tg_end": 2015,
+      "prod_state_year_tg_start": 2013,
+      "prod_traj_year_final": 2015,
+      "prod_traj_year_initial": 2001,
+      "task_name": "",
+      "task_notes": "",
+      "trajectory_method": "ndvi_trend"
+    },
+    "progress": 0,
+    "results": {},
+    "script_id": "47b99c3d-f5b5-4df3-81e7-324a17a6e147",
```

Comment:
script_id needs to be kept in sync with latest value taken from [`scripts.json`](https://github.com/ConservationInternational/trends.earth/blob/main/LDMP/data/scripts.json#L132). These are kept in sync by [an invoke task](https://github.com/ConservationInternational/trends.earth/blob/main/tasks.py#L502). Script IDs change when there are upstream changes in trends.earth-algorithms or trends.earth-schemas, or when the GEE code used on the API changes